main.go

```diff
@@ -0,0 +1,76 @@
+package main
+
+import (
+	"encoding/csv"
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+
+	"github.com/gorilla/mux"
+)
+
+type Employee struct {

```

Comment:
By convention, all the descriptions should start with the name of the struct, interface, function, etc name
`// Employee - Description`